epoch/api/channels_api_usage.md

```diff
@@ -339,15 +342,246 @@ it and then post it back via a WebSocket message:
+it and then post it back via a WebSocket message:
+```
+{'action': 'update_ack',
+ 'payload': {
+    'tx': 'tx$C4TACwXiGbsZg1Z9ChAEuaVJge62Uax5riSw4Tpp763SKTwLLGjhqPDUqjb9swx5X2G6XnpQk72v1yhFSLZvPtUv4JgoimKLqhAVUSoHB3btwEUTdgb3kVbaS6WLiaF86cqBZiDwbiL3GpThgZK7VhYedXFZK6PdK2RjywURhC9s8PNwP19Zetj19VkL4MYn1Z5tkoEh387dMfh9Mfh9jPf7gaBJjvwxNLuMjqHQXwSKPEpR8jYBF4cuzwaZzWeUSPKMxp44wDWQqqG8MTMwR1i8JbZjbAFtuj49TJWMw6xrvXski8XieHLHXrx5y4873CgD1FhRQT374Yvb5mYBTMtsYc8ttUCPiwi7AQteey7h1iRJgGc5bcd2omdGG8BmQZETfLaqwqXWN7mXXxPGfJQUaknHdocTum9FNyncYiAt6Y7znFJ1rZT2UkHvKwbvtJvhftWBJPrbExfeFMypn9DoW4SoaQ7RjBcmz5PWdt79BjRXhZpM7qbjvGF9i1f2cXiQFeKcj9FxXdzttN28kdtDvpYWdBbWcSMaWAYy7EBnMRHrGo2XdcDeJeGQdGnSBKaMHdN5LVhKF'
+  }
+}
+```
+#### Finish update
+After both the parties had signed the new updated state of the channel - it is
+considered the latest one. Corresponding update messages are sent to both
+parties to indicate it. In the payload the message there is the latest
+co-signed off-chain update so the participants can persist it locally.
+
+```
+{'action': 'update',
+ 'payload': {
+    'state': 'tx$L1mTJpyV97nWgMAFmrqcwECSThovgaU9qv5DYeqn3YQgzjCShS96n6u2rACsXyXAJyRrqGNTv2ytC1LgmH8zSuD6SkDTEiT2eCGuCsM1xXyZGooq14nu8naSZ3rhJEwBoZLTn3Tt86W3eUc1rPs2HaFwtxwsFhFu4dXHJTjmaVjigG8Juhvq1oQytscRCPVt9sjijr8h1mCCj73yhnsvwwJuMLi9WgspqwJfHHRVc8RWPtvKg6hbUKxn4cEP2M1rpXhVEUhCAkgKVfm58m4xvnuCNhvJhi37cGeSMSJkdrs64111jnQY2nwpiiS5dRW8c3VcHt1Zm8Sg1fyQgdmgbr3Rwbez1L6hCndppgX1DA2UniQB1y7eqC8nYMe4DUmXxnjcFxCQyaiTCBGavjcZiQUThHzwoACJ1yf4fr1HXZ2BzsgeSsuE2AuvphAvtiPSDmRUiFwxt1KGg3YRBxsSqs2Ums7D6QDoju11VH3k8ERcb9Hj6uJmHFC2yGu48jKHPB8bUCm1qDBZqEiDBVTQo6E1AiSLtvJmLoRd1gyUZ4pvCPPmGPQJ4Qsb16xH6q4YnJVMVQb7TWSs5PorsdqV1XPKg8rZSwSZJGZ1gT7fkf75ocXt986k3PMNCa98cDQmwbqFuqYYC4iMixj6k2txsFYUCyxrucHidNC8MFC'}
+ }
+```
+
+#### Getting a call result
+All calls are stored in the channel state tree. In order to extract one out of
+there and inspect it, one shall send a WebSocket event
+```
+{'action': 'get',
+ 'tag': 'contract_call',
+ 'payload': {
+    'contract_address': 'ct$9sRA9AVE4BYTAkh5RNfJYmwQe1NZ4MErasQLXZkFWG43TPBqa',
+    'caller_address': 'ak$2YeNqB4jQ1DG7QvUKgCkKeZAiXb2rnzkEoLTS8XeKF8Smgit2e',
+    'round': 8
+		}
+}
+```
+The `contract_address` is the address of the contract that had been called, the `round` is the round of the update and
+`caller_address` is the address of the caller.
+
+Then the call is returned through an incoming message:
+```
+{'action': 'get',
+ 'tag': 'contract_call',
+ 'payload': {
+      'contract_address': 'ct$9sRA9AVE4BYTAkh5RNfJYmwQe1NZ4MErasQLXZkFWG43TPBqa',
+			'caller_address': 'ak$2YeNqB4jQ1DG7QvUKgCkKeZAiXb2rnzkEoLTS8XeKF8Smgit2e',
+      'caller_nonce': 8,
+      'gas_price': 0,
+			'gas_used': 524,
+      'height': 8,
+      'return_type': 'ok',
+      'return_value': '0x000000000000000000000000000000000000000000000000000000000000002A'
+		}
+}
+```
+
+It is worth mentioning that since this is an off-chain transaction the gas price specified is not consumed.
```

Comment:
I've added the following just bellow this line:
>That amount of gas represents the amount of computations. It could be used for
aproximation for the gas needed for executing a contract on-chain if a similar amount of
computations are required. Computation heavy contracts might be just too
expensive to be force progressed on-chain, so please use with caution.

Reply:
Ideally that should go into the protocol section and not just the usage but that can be changed later